signer/src/transaction_signer.rs

```diff
@@ -277,6 +277,8 @@ where
             (message::Payload::BitcoinPreSignRequest(requests), _, _) => {
                 self.handle_bitcoin_pre_sign_request(requests, &msg.bitcoin_chain_tip)
                     .await?;
+                self.context
+                    .signal(TxSignerEvent::BitcoinPreSignRequestReceived.into())?;
```

Comment:
If we send a pre-sign ack message then we should remove this.

Reply:
removed in https://github.com/stacks-network/sbtc/pull/956/commits/eec1c22ee13bac3395a6568823e6bc7f8cd8621f